src/test/rgw/sfs/test_rgw_sfs_sfs_bucket.cc

```diff
@@ -1453,3 +1497,191 @@ TEST_F(TestSFSBucket, ListNamespaceMultipartsBasics) {
   EXPECT_EQ(results.objs[0].key.name, std::to_string(id));
   EXPECT_EQ(results.objs[0].meta.mtime, now);
 }
+
+TEST_F(TestSFSBucket, RacedBucketMetadataWriteOperations) {
+  auto ceph_context = std::make_shared<CephContext>(CEPH_ENTITY_TYPE_CLIENT);
+  ceph_context->_conf.set_val("rgw_sfs_data_path", getTestDir());
+  ceph_context->_log->start();
+  auto store = new rgw::sal::SFStore(ceph_context.get(), getTestDir());
```

Comment:
test dir cleanup code missing

Reply:
will fix

Reply:
Apologies, what do you mean with that?

Reply:
The code above creates a new store at getTestDir, but never deletes it. See the other unit tests teardown hooks, there is usually a delete in there.

Reply:
but isn't this function supposed to be called for every test:
`void TearDown() override` ?
am I missing something here?